admin/views/js-templates-primary-term.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * @package WPSEO\Admin
+ */
+
+if ( ! defined( 'WPSEO_VERSION' ) ) {
+	header( 'Status: 403 Forbidden' );
+	header( 'HTTP/1.1 403 Forbidden' );
+	exit();
+}
+?>
+
+<script type="text/html" id="tmpl-primary-term-input">
+	<input type="hidden" class="yoast-wpseo-primary-term"
+	       id="yoast-wpseo-primary-{{data.taxonomy.name}}"
+	       name="<?php echo WPSEO_Meta::$form_prefix; ?>primary_{{data.taxonomy.name}}_term"
+	       value="{{data.taxonomy.primary}}">
+
+	<?php wp_nonce_field( 'save-primary-term', WPSEO_Meta::$form_prefix . 'primary_{{data.taxonomy.name}}_nonce' ); ?>
+</script>
+
+<script type="text/html" id="tmpl-primary-term-ui">
+	<a href="#" class="wpseo-make-primary-term"><?php
```

Comment:
Does this `href` get populated with something later? If not, then a `<button>` (style like a link if you wish) is more semantically correct and more accessible.